File: src/main/java/com/procurement/notice/model/entity/BudgetEntity.java
package com.procurement.notice.model.entity;

import java.util.Date;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class BudgetEntity {

    private String cpId;

    private String ocId;

    private Date releaseDate;

    private String releaseId;

    private String stage;

    private Double amount;

    private String jsonData;
}

File: src/main/kotlin/com/procurement/notice/model/entity/BudgetEntity.kt
package com.procurement.notice.model.entity

import com.fasterxml.jackson.annotation.JsonInclude
import java.math.BigDecimal
import java.util.*

data class BudgetEntity(

    var cpId: String,

    var ocId: String,

    var publishDate: Date,

    var releaseDate: Date,

    var releaseId: String,

    var stage: String,

    @JsonInclude(JsonInclude.Include.NON_NULL)
    var amount: BigDecimal? = BigDecimal.valueOf(0.00),

    var jsonData: String

)


